apps/core/scenes/search/Placepickers.js

```diff
@@ -24,7 +24,7 @@ import { MODAL_TYPE } from './SearchConstants';
 import PickersWrapper from './PickersWrapper';
 
 type Props = {|
-  +travelFrom: ?Location,
+  +travelFrom: ?Array<Location>,
   +travelTo: ?Location,
```

Comment:
Why is it only for `travelFrom` and not `travelTo`?

Reply:
nice shot I'll check why flow didn't warn me thanks.